modules/ROOT/pages/runtime-installation-task.adoc

```diff
@@ -84,28 +84,15 @@ $ $MULE_HOME/bin/mule install
 
 == Configure Properties in the wrapper.conf File
 
-After installing Mule as a Windows service, you can override the properties defined in the `wrapper.conf` file by declaring new properties in the `wrapper.additional.conf` file.
+After installing Mule as a service/daemon, you can modify the properties defined in the `wrapper.conf` file.
```

Comment:
```suggestion
After installing Mule as a service daemon, you can modify the properties defined in the `wrapper.conf` file.
```

Reply:
I used the slash because they are two different terms for something very similar (typically `service` is used in Windows environments and `daemon` is used for Unix environments). They both refer to something that is constantly running in the background.
It is also reference like that [here](https://github.com/mulesoft/docs-mule-runtime/blob/926a02af3879c231aff7547727426832a6c31da6/modules/ROOT/pages/runtime-installation-task.adoc?plain=1#L69) and [here](https://github.com/mulesoft/docs-mule-runtime/blob/926a02af3879c231aff7547727426832a6c31da6/modules/ROOT/pages/runtime-installation-task.adoc?plain=1#L109).